plugins/providers/virtualbox/driver/version_5_0.rb

```diff
@@ -17,19 +17,23 @@ def initialize(uuid)
         end
 
         def clear_forwarded_ports
-          args = []
-          read_forwarded_ports(@uuid).each do |nic, name, _, _|
-            args.concat(["--natpf#{nic}", "delete", name])
-          end
+          retryable(on: Vagrant::Errors::VBoxManageError, tries: 3, sleep: 1) do
```

Comment:
The `args` could become invalid after the first retry, so retrying the whole block makes more sense. Same pattern follows in the rest of this. Nevermind :D LGTM!